.github/workflows/cluster-test/mysql/dolphinscheduler_env.sh

```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 #
 
+# Load system configuration
+source /etc/profile
```

Comment:
I think it's not a good idea since this may destroy the user's environment variables. cc @EricGao888 @zhongjiajie @ruanwenjun

Reply:
I don't like the idea to `source /etc/profile` either. Some users may put their real `profile` files in a different path, in that case, `source /etc/profile` may cause some big trouble.

Reply:
> I think it's not a good idea since this may destroy the user's environment variables. cc @EricGao888 @zhongjiajie @ruanwenjun

The user environment variable is configured after this command, and the increment or update configuration operation will still follow the user configuration requirements.

Reply:
> I don't like the idea to `source /etc/profile` either. Some users may put their real `profile` files in a different path, in that case, `source /etc/profile` may cause some big trouble.

The default path is / etc / profile. The user needs to change the corresponding source file address when storing in other places. Therefore, relevant comments are given to load the system environment variable configuration

Reply:
> > I think it's not a good idea since this may destroy the user's environment variables. cc @EricGao888 @zhongjiajie @ruanwenjun
> 
> The user environment variable is configured after this command, and the increment or update configuration operation will still follow the user configuration requirements.

If user want to customize environment variables. they can add them in `dolphinscheduler_env.sh` or in the environment management of the UI interface. However, we do not recommend hardcode the `source /etc/profile`.

Reply:
> 

After testing, I found that this did not affect

Reply:
But there is a bug:
`export JAVA_HOME=${JAVA_HOME:-/usr/local/openjdk-8}`
We want to use the system environment variables, but it does not work in reality.
How we can get the system environment without `source /etc/profile` ?

Reply:
For example. In AWS EMR, if a non root user and a non system installation user such as `ubuntu/hadoop/ec2-user` are used to execute the `source /etc/profile` command. It will directly lead to the `hive/flink/spark` cli no such file exception.

Reply:
@SbloodyS 
So, what's your suggestions about this issue?

Reply:
> @SbloodyS So, what's your suggestions about this issue?

I think the current `export JAVA_HOME=${JAVA_HOME:-/usr/local/openjdk-8}` a is a good solution for manual configuration. If the user's image does not contain global `JAVA_HOME`. It should only be configured manually by the user or find the operations engineer to optimize the basic image.

Reply:
> > @SbloodyS So, what's your suggestions about this issue?
> 
> I think the current a is a good solution for manual configuration. If the user's image does not contain global . It should only be configured manually by the user or find the operations engineer to optimize the basic image.`export JAVA_HOME=${JAVA_HOME:-/usr/local/openjdk-8}``JAVA_HOME`

In order to obtain system variables, source / etc / profile can solve this problem. Export Java_ Home = ${java_home: - / usr / local / openjdk-8} after source, which means that when Java does not exist in the system variable_ When home, the specified file is used. If the system variable has been configured, the system configured variable is used.

Reply:
i think so, and the environment variables do not only exist in the /etc/profile file, I think the specificity of the worker host should be ignored, which can enhance the user's autonomy